site/docs/components/spinner/accessibility.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Spinner
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Best practises
+
+- Improve accessibility by customizing the aria-label to provide additional context about what is loading, for example, `aria-label="loading settings panel"`.
+
+- The aria-label will be announced to screen-reader users at a regular interval which can be customised using the `announcerInterval` property.
+
+- After a set timeout the component will stop announcing the aria-label, this can be customised using the `announcerTimeout` property
+
+- A completion message will be announced to screen reader users when the spinner is unmounted to indicate that loading had completed. This can be customised using the `completionAnnouncement` prop or disabled by setting `completionAnnouncement={null}`
```

Comment:
```suggestion
- Improve accessibility by customizing the `aria-label` to provide additional context about what is loading, for example, `aria-label="loading settings panel"`.

- The `aria-label` will be announced to screen-reader users at a regular interval which can be customised using the `announcerInterval` property.

- After a set timeout the component will stop announcing the `aria-label`, this can be customised using the `announcerTimeout` property

- A completion message will be announced to screen reader users when the spinner is unmounted to indicate that loading had completed. This can be customised using the `completionAnnouncement` prop or disabled by setting `completionAnnouncement={null}`
```